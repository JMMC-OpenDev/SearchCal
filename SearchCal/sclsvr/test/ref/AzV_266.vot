version https://git-lfs.github.com/spec/v1
oid sha256:a73ec7c1ca16f08068c01fa1eaccc1e998e51ebd13fa511249bed84dc51db610
size 42512
