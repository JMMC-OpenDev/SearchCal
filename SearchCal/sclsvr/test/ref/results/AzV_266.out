version https://git-lfs.github.com/spec/v1
oid sha256:ccb48a74a71aa5c5006ae84ca8927a83925397d67974487976b1f5d6b62c9a70
size 368975
